### PR TITLE
10.0 pos invoice mobile

### PR DESCRIPTION
**Pull request by KolushovAlexandr**

### CODE REVIEW

File: pos_mobile/static/src/xml/pos.xml [hunk 1]
```diff
@@ -45,12 +45,16 @@
     </t>
 
     <t t-extend="PaymentScreenWidget">
-        <t t-jquery="div[class='paymentmethods-container']">
-            $(this).addClass('touch-scrollable scrollable-y');
+        <t t-jquery="div[class='paymentmethods-container']" t-operation="replace">
+        </t>
```
Comment (GabbasovDinar) [inline pos_mobile/static/src/xml/pos.xml:49]: don't replace this element

File: pos_invoice_pay/static/src/js/main.js [hunk 2]
```diff
@@ -870,7 +870,11 @@ var InvoicesWidget = InvoicesAndOrdersBaseWidget.extend({
 gui.define_screen({name:'invoices_list', widget: InvoicesWidget});
 
 var InvoicePayment = screens.PaymentScreenWidget.extend({
```
Comment (GabbasovDinar) [inline pos_invoice_pay/static/src/js/main.js:872]: why not "include" instead of "extend"?

**Review by GabbasovDinar — CHANGES_REQUESTED** [review]

**Comment by yelizariev** [comment]
@KolushovAlexandr

**Comment by yelizariev** [comment]
WIP

**Comment by GabbasovDinar** [comment]
fixes here: https://github.com/it-projects-llc/pos-addons/pull/833

**Pull request closed without merging**